build.py

```diff
@@ -17,7 +17,7 @@ def run_python(cmd):
         cmd = Path(__file__).with_name(cmd)
         if not cmd.is_file():
             raise UserWarning(f'no such file: {original}')
-    return run_path(cmd)
+    return run_path(str(cmd))
```

Comment:
Hm, I was sure that I fixed similar issue previously..
Could you provide sample case to reproduce this bug?

Reply:
Yes, just add these lines to build:
```
        'post_build': [
            'web_meta_gen.py'
        ]
```
using python3.8, it causes an error:
```
Traceback (most recent call last):
  File "build.py", line 171, in <module>
    backup_and_run()
  File "build.py", line 87, in backup_and_run
    iitc_build(source, workdir, deps_list=deps_list)
  File "build.py", line 65, in iitc_build
    run_cmds(settings.post_build, source, outdir)
  File "build.py", line 38, in run_cmds
    module = run_python(Path(cmd))
  File "build.py", line 20, in run_python
    return run_path(cmd)
  File "/usr/lib/python3.8/runpy.py", line 262, in run_path
    code, fname = _get_code_from_file(run_name, path_name)
  File "/usr/lib/python3.8/runpy.py", line 232, in _get_code_from_file
    with io.open_code(fname) as f:
TypeError: open_code() argument 'path' must be str, not PosixPath
```

Reply:
Well, it's weird.
I have Python 3.8.
And have these lines in `localbuildsettings.py`:
```py
        'post_build': [
            'web_meta_gen.py',
```
And when I run `build.py` I see no problems:
```
IITC build: local
Generating meta for build: local
```